src/Input/KeyInterop.ts

```diff
@@ -170,8 +170,9 @@ module Fayde.Input {
             }
             return args;
         }
+		
         CreateArgsDown(e): Fayde.Input.KeyEventArgs {
-            if (e["char"] && e.keyCode !== 8 && e.keyCode !== 9)
+            if (e["char"] && e.keyCode !== 8 && e.keyCode !== 9 && !e.ctrlKey)

```

Comment:
There are certain key presses that make it through `down' and 'press'.  In essence, it looks like a venn diagram which events they trigger.  The intention In the 'down' event is to filter out keys that happen in 'press'.  If these keys aren't flowing through 'press', the problem could be resolved directly.